### DISCUSSION TITLE
Using Downloaded Datasets

### DISCUSSION THREAD

**Discussion by GoudaCouda** [erew123/alltalk_tts#426] [General]
How can I import a data set from somewhere else. I am aware I can put the audio and press generate dataset but how can I create the spreadsheets when I already have all my clips split up already?

**Comment by erew123** [comment]
@GoudaCouda From within the Finetuning interface, please see the "Using your own training dataset" section

![image](https://github.com/user-attachments/assets/7f63f81f-e4de-4573-8846-ff75ca0189c5)

**Comment by erew123** [comment]
@GoudaCouda Ive put this together for you https://github.com/erew123/alltalk_tts/wiki/XTTS-Finetuning-Manually-Creating-Your-Own-Dataset

Its a little more detailed than what is in the Finetuning interface
